.github/workflows/cd-analytics-infra.yml

```
@@ -0,0 +1,61 @@
+name: Deploy Analytics Infra
+# Todo: create a composite action
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "infra/analytics/**"
```

```suggestion
    paths:
      - "infra/analytics/**"
  release:
    types: [published]      
```

This file needs to deploy prod on releases as well. So do the other files